tests/unit/storage_v3_query_plan_accumulate_aggregate.cpp

```diff
@@ -0,0 +1,634 @@
+// Copyright 2022 Memgraph Ltd.
```

Comment:
Could you please explain why these tests has `storage_v3` in their name but using `storage-v2`?

Reply:
Because the schema is currently in storage-v2 and not in v3, and that will be all moved in the next PR, both query and storage to their next version, since the schema epic began before storage-v3 was introduced